.ci/jenkins/base_image.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy=${http_proxy} https_proxy=${https_proxy}
```

Comment:
to be sure I would export also uppercase
https://unix.stackexchange.com/questions/212894/whats-the-right-format-for-the-http-proxy-environment-variable-caps-or-no-ca

Reply:
done